src/websocket_server/docker_entrypoint.sh

```
@@ -1,6 +1,7 @@
 #!/bin/sh
 OM_HOST="${OMNIBUS_SERVER_HOST:-localhost}"
 echo "Omnibus Server: $OM_HOST"
-exec uv run --no-sync ./src/websocket_server/main.py "$@" <<EOF
+cd ./src/websocket_server
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)